sampling/include/var_opt_union_impl.hpp

```diff
@@ -437,7 +437,7 @@ bool var_opt_union<T,S,A>::detect_and_handle_subcase_of_pseudo_exact(var_opt_ske
   const bool condition2 = gadget_.num_marks_in_h_ > 0;
 
   // if gadget is pseudo-exact and the number of marks equals outer_tau_denom, then we can deduce
-  // from the bookkeeping logic of mergeInto() that all estimation mode input sketches must
+  // from the bookkeeping logic of merge_nto() that all estimation mode input sketches must
```

Comment:
perhaps, merge_into()

Reply:
oops.. typo when updating the method name